src/theme/Footer/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import styles from './styles.module.css';
+
+const content = {
+  sponsorsContent: (
+    <>
+      If you like our work, you can <b>sponsor & support</b> the community!<br></br>
+      All the money will be used to continue our mission of <b>providing simplified cloud native education to all</b> and make this community reach more number of people.
+    </>
+  )
+}
+
+const logo = {
+  hashnode: { 
+    Svg: require('@site/static/img/Vector.svg').default
+  },
+  coffee : {
+    Svg: require('@site/static/img/coffee.svg').default
+  },
+  mail: {
+    Svg: require('@site/static/img/mail.svg').default
+  },
+  instagram: {
+    Svg: require('@site/static/img/instagram.svg').default
+  },
+  twitter: {
+    Svg: require('@site/static/img/twitter.svg').default
+  },
+  github: {
+    Svg: require('@site/static/img/github.svg').default
+  },
+  youtube: {
+    Svg: require('@site/static/img/youtube.svg').default
+  },
+  linkedin: {
+    Svg: require('@site/static/img/linkedIn.svg').default
+  }
+}
+
+function Footer() {
+  return (
+    <section className={styles.footer}>
+      <div>
+        <div className={styles.title}>help us do more</div>
+        <div className={styles.sponsors_content}>{content.sponsorsContent}</div>
+        <div className={styles.sponsors_card}>
+          <div className={styles.sponsors_card_title}>Sponsor us on Hashnode</div>
+          <button className={styles.sponsor_hashnode}>
+            <div className={styles.sponsor_hashnode_logo}>
+              <logo.hashnode.Svg className={styles.logo} role="img" />
+              <div className={styles.button_name}>Sponsor</div>
+            </div>
+          </button>
+          <div className={styles.sponsors_or}>or</div>
+          <button className={styles.sponsor_coffee}>
+              <logo.coffee.Svg className={styles.coffee} role="img" />  
+          </button>
+        </div>
+      </div>
+      <hr className={styles.hr}></hr>
+      <div className={styles.end_first}>© 2022 Kubesimplify</div>
+      <logo.mail.Svg className={styles.mail_logo} row="img" />
+      <div className={styles.mail_title}>example@kubesimplify.com</div>
+      <div className={styles.group_logo}>
+        <logo.twitter.Svg row="img" />
+        <logo.instagram.Svg row="img" />
+        <logo.github.Svg row="img" />
+        <logo.linkedin.Svg row="img" />
+        <logo.youtube.Svg row="img" /> 
```

Comment:
Add redirect links with the target as blank to the logos to different kubesimplify socials